homeassistant/components/plex/__init__.py

```diff
@@ -75,7 +75,11 @@
     )
 )
 
-CONFIG_SCHEMA = vol.Schema({PLEX_DOMAIN: SERVER_CONFIG_SCHEMA}, extra=vol.ALLOW_EXTRA)
+CONFIG_SCHEMA = vol.Schema(
+    cv.deprecated(PLEX_DOMAIN, invalidation_version="0.111"),
```

Comment:
We're missing a `vol.All` to wrap the deprecation and the rest of the schema.

Reply:
Good catch. Fixed in #34552.